modules/nf-core/graphtyper/vcfconcatenate/main.nf

```diff
@@ -0,0 +1,36 @@
+process GRAPHTYPER_VCFCONCATENATE {
+    tag "$meta.id"
+    label 'process_low'
+
+    conda "bioconda::graphtyper=2.7.2"
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/graphtyper:2.7.2--h7d7f7ad_0':
+        'quay.io/biocontainers/graphtyper:2.7.2--h7d7f7ad_0' }"
+
+    input:
+    tuple val(meta), path(vcf)
+
+    output:
+    tuple val(meta), path("*.concatenated.vcf.gz"), emit: vcf
```

Comment:
```suggestion
    tuple val(meta), path("*.vcf.gz"), emit: vcf
```

It's preffered to not force extra extensions on the pipeline developers (they need to add it themselves using `prefix`). You can always add a check for this like shown here: https://nf-co.re/docs/contributing/modules#naming-conventions paragraph 6